package.json

```
@@ -1,4 +1,5 @@
 {
+  "name": "@crm/source",
```

Biome (1.9.4)

[error] The key name was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)